doc/guide/Guide_CLIMADA_Tutorial.ipynb

```diff
@@ -118,7 +118,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.5"
+   "version": "3.8.8"
```

Comment:
I don't know why the version changed, and if it matters?

Reply:
I do not know either, but I do not think so. If I understand it correctly, this is simply the python version you have used while making the notebook.